src/lib.rs

```diff
@@ -1503,3 +1516,7 @@ impl<T: Counter, F: Counter> PartialEq<Histogram<F>> for Histogram<T>
 #[path = "tests/tests.rs"]
 #[cfg(test)]
 mod tests;
+
+/// Serialization/deserialization support.
+#[path = "serialization/serialization.rs"]
```

Comment:
Because that way leads to a whole bunch of `mod.rs`, which I find distasteful. It makes it more difficult or clumsy to glance at a filename and know what it is, or jump to the file you want.